models/ga4_export__conversions_report.sql

```diff
@@ -1,20 +1,11 @@
 {{
     config(
-        enabled=var('key_events', none) is not none and var('key_events') != []
-    )
-}}
-
-{{
-    config(
+        enabled=var('key_events',[]) != [],
         materialized='incremental' if ga4_export.is_incremental_compatible() else 'table',
         unique_key='unique_key',
         incremental_strategy='insert_overwrite' if target.type in ('bigquery', 'spark', 'databricks') else 'delete+insert',
-        partition_by={
-            "field": "event_date", 
-            "data_type": "date"
-            } if target.type not in ('spark','databricks') 
-            else ['event_date'],
-        cluster_by=['event_name', 'event_date'],
+        partition_by={"field": "event_date", "data_type": "date"} if target.type not in ('spark','databricks') else ['event_date'],
+        cluster_by=['event_name'],
```

Comment:
Clustering helps organize the partitions. There is no need to cluster by the partitioned field. It's duplicative so we can remove this for each incremental model.